### PR TITLE
Form create ambassador

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by TheoBoschetti — DISMISSED** [review]
seems legit

File: frontend/src/components/FormCreateAmbass.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        alert(res.data);
+      })
+      .catch(err => {
+        alert(err);
+      });
+  };
+
+  return (
+    <div className="FormCreateAmbass">
+      <h1>Create new ambass</h1>
```
Comment (LoicBrassart) [inline frontend/src/components/FormCreateAmbass.jsx]: Maybe finish your words ?

**Review by LoicBrassart — DISMISSED** [review]
There's a typo, I hope you'll fix it

File: frontend/src/components/FormCreateAmbass.jsx (hunk 1 above)
Comment (LeoDiduch) [reply frontend/src/components/FormCreateAmbass.jsx]: changed

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Review by LoicBrassart — APPROVED** [review]
Go!

**Pull request merged**